.github/workflows/maven.yml

```diff
@@ -17,11 +17,13 @@ jobs:
         uses: actions/checkout@v3
         with:
           lfs: false
+          fetch-depth: 0  # Needed for cldr-archive
```

Comment:
I'm not sure what this means...

I found this, not sure what the implication is:
https://github.com/changesets/action/issues/97 "This is no longer necessary: the tool will automatically deepen shallow clones as necessary"

Reply:
I think that's for an unrelated tool. cldr-archive does _not_ automatically deepen shallow clones as necessary.

Reply:
good to know! so I'm guessing zero means no limit